.github/workflows/scorecard.yml

```diff
@@ -32,7 +33,7 @@ jobs:
 
     steps:
       - name: "Checkout code"
-        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+        uses: actions/checkout@4
```

Comment:
Using the exact commit hash is done intentionally.

It's a suggested security measure, advised by security scorecard.

Reply:
We can pin them to hash, but there would be better to enable @renovate-bot for this repo to keep thing up to date.

Reply:
Dependabot can assist here as well I believe